doc/rados/operations/pools.rst

```diff
@@ -378,45 +381,41 @@ You may set values for the following keys:
 
 .. describe:: pg_num
 
-   The effective number of placement groups to use when calculating
+   Sets the effective number of PGs to use when calculating
    data placement.
 
    :Type: Integer
-   :Valid Range: Superior to ``pg_num`` current value.
+   :Valid Range: Greater than the current value of ``pg_num``.
 
 .. _pgp_num:
 
 .. describe:: pgp_num
 
-   The effective number of placement groups for placement to use
-   when calculating data placement.
+   Sets the effective number of PGs to use when calculating data placement.
 
    :Type: Integer
-   :Valid Range: Equal to or less than ``pg_num``.
+   :Valid Range: Less than or equal to the current value of ``pg_num``.
 
 .. _crush_rule:
 
 .. describe:: crush_rule
 
-   The rule to use for mapping object placement in the cluster.
+   Sets the rule that Ceph uses to map object placement in the cluster.
```

Comment:
The CRUSH rule that Ceph uses to determine RADOS object placement within the pool.

Reply:
Accepted.